packages/reason-bindings/examples/example_configure_flopflip.re

```diff
@@ -1,4 +1,4 @@
-open FlopFlipTypes;
+open FlopFlip__Types;
 
```

Comment:
Silly question: is the double underscore intentional?

Reply:
Yes, apparently is some kind of convention for namespacing